cumulusci/tasks/push/pushfails.py

```diff
@@ -0,0 +1,69 @@
+""" simple task(s) for reporting on push upgrade jobs.
+
+this doesn't use the nearby push_api module, and was just a quick ccistyle
+get the job done kinda moment.
+"""
+
+import re
+import csv
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class PushFailTask(BaseSalesforceApiTask):
```

Comment:
How about `ReportPushFailures`. Elsewhere we haven't used the Task suffix (tasks are usually addressed via the `tasks` package so it seems redundant) but I don't have an actual problem with doing so.